packages/web-pkg/src/composables/actions/files/useFileActionsCreateNewFolder.ts

```diff
@@ -76,6 +76,10 @@ export const useFileActionsCreateNewFolder = ({ space }: { space?: SpaceResource
         })
       }
 
+      if (isShareSpaceResource(space)) {
+        resource.remoteItemId = space.id
```

Comment:
we can't do this, when we build the resource ?

Reply:
Because we don't have the space in that context. It would be great to add it in the future, but that needs more refactoring.

Reply:
Fine for now, could you add a fixme or todo comment anyways ?